Segments/Blackburrow.mapproj

```diff
@@ -1061,44 +1032,26 @@ public static bool spawnCounter = true;
         public override void OnHit(MobileEntity attacker, MobileEntity defender)
         {
         
-            var distance = GetDistanceToMax(defender.Location);
-            var chargeDisplay = (ChargesCurrent-1).Clamp(0,3);
-        
-            if (attacker is PlayerEntity player)
-          {
-
-            trackHits++;
-            var packet2 = new AsciiMessagePacket(AsciiMessageType.System, Color.Red, 
-                $"The Pike has {trackHits} hits, and {chargeDisplay} charges.");
-                attacker.SendPacket(packet2);
-            
-
-            if (trackHits >= 6)
+            var cleaveVictim = attacker.GetBeheldInVisibility(0).OfType<CreatureGroup>().SelectMany(m => m.Members);
+            var meleeDamage = (double)attacker.CalculateMeleeDamage(this, defender);
+            var cleaveDamage = meleeDamage * .05;
 
+            if (cleaveVictim != null)
             {
-                
-                var packet = new AsciiMessagePacket(AsciiMessageType.System, Color.Red, 
-                $"Your Pike crackles and begins to feel weightless in your hand!");
-                attacker.SendPacket(packet);
-                attacker.PlaySound(67);
-                
-                if (ChargesCurrent < 4)
-                {
-                     ChargesCurrent++;
-                }
+              foreach (var victim in cleaveVictim)
+             {
+                 var distance = attacker.GetDistanceToMax(victim.Location);
+
+                 if (distance == 0)
+                 {
+                   if (victim != defender && victim.IsAlive)
+                   victim.ApplyDamage(attacker, (int)cleaveDamage);
```

Comment:
Just trying to make sure here on the usage. For venom I had to reverse entities. 

![image](https://github.com/jkachhad/Stormhalter/assets/90510109/15379870-d0e3-4f03-84b7-36bd50f2d763)